src/sender.rs

```diff
@@ -32,11 +31,12 @@ where
     T: Serialize + Deserialize<'de>,
 {
     fn clone(&self) -> Sender<T> {
+        use std::sync::Arc;
```

Comment:
If we use previous syntax, all these `use std::sync::Arc` will be not needed

Reply:
We could also just `use std::sync` at a global level instead. No compelling reason to do this over the global import.